app/assets/stylesheets/components/_jumbotron.scss

```diff
@@ -1,9 +1,10 @@
 .jumbotron {
   background-image:
-  linear-gradient(to bottom, rgba(245, 246, 252, 0.4), rgba(144, 246, 168, 0.5)),
+  linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
```

Comment:
Is this a linear gradient or just a box with transparency??

Reply:
It is sort of a transparent black overlay